resources/js/app.js

```diff
@@ -137,13 +183,46 @@
         }
     }]);
 
-    app.directive('faq',[ '$http', function ($http) {
+    app.directive('faq',[ '$http', '$templateCache', function ($http, $templateCache) {
         return {
             restrict: 'E',
             templateUrl: '/partials/tabs/faq.html',
             controller: function () {
                 self = this
+                lang = window.localStorage.getItem('lang')
                 self.faqs = faq
+
+                function setFromDefault(key) {
```

Comment:
here as well, if you feel its only a part of this controller an won't reuse this function, self.setFromDefault will be better. If a reuse is a possibility, then I suggest using Factory